src/frontend-pwa/src/components/appNav/SmallNavButton/SmallNavButton.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @summary A small Nav button intended for the footer when in Mobile view
+ * @param path The route to travel to
+ * @param icon an svg image to use for the icon
+ * @param text informative text for below the icon
+ * @author LocalNewsTV
+ */
+import { Link } from 'react-router-dom';
+import { ButtonText, IconWrapper, InputWrapper } from './SmallNavButton.styles';
```

Comment:
I believe the import path should be lowercase 's'. As in './smallNavButton.styles'.